### PR TITLE
Refactor with Hooks

### PR DESCRIPTION
**Pull request by Epaltechs**
### At this point in the project, I:
- Refactor the Calculator component (components/Calculator.js) from a class-based component to a functional component with hooks.
- Ensure the Calculator functionality is maintained.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,39 +1,26 @@
...
+  const [state, setState] = useState(props.obj);
+  return (
+    <div className="main-container">
+      <div className="console">{state.total}{state.operation}{state.next}</div>
+      {props.data.map((i) => {
+        return <button onClick={()=>
+          setState(calculate(state, i))} key={(count += 1)}>{i}</button>
+      })}
+    </div>
+  );
```
Comment (Cathella) [inline src/components/Calculator.js:16]: ✅ The Calculator component (components/Calculator.js) should be refactored from a class-based component to a functional component 👍🏻

File: README.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 "Math Magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to Make simple calculations, read a random math-related quote.
 
+![Screenshot (77)](![screenshot](./src/screen.png))
```
Comment (Cathella) [inline README.md]: - [ ] [OPTIONAL] Fix the screenshot to clearly show up in the README.

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @Epaltechs 👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💪🏻

✅ No linter errors.
✅ You used correct GitHub Flow.
✅ Professional README.
✅ App still works as expected.


Cheers and Happy coding!👏👏👏

thats-how-its-done-allison-janney


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Epaltechs) [reply README.md]: I will, many thanks.

**Pull request merged**